chrome-app/links.js

```diff
@@ -30,6 +30,7 @@ const LINKS = [
         ))
     ],
     [
+        { title: "Small", href: gitHubQuery('https://github.com/issues', [...issue, ...ourRepos, "assignee:USERNAME", "label:bug", "-label:fix-ready", "label:effort/small" ] ) },
```

Comment:
+1 to this one. Although it depends on having labeled issues as `effort/small`. I haven't done that to any of them... :x

Reply:
Now that I have a quicklist, I've started to do that, and it's awesome!